src/Store/VimStoreConnector.re

```diff
@@ -121,7 +121,7 @@ let start =
         | Info => ("INFO", Notification.Info)
         };
 
-      Log.infof(m => m("Message - %s [%s]: %s", priorityString, t, msg));
+      Log.debugf(m => m("Message - %s [%s]: %s", priorityString, t, msg));
 
       dispatch(ShowNotification(Notification.create(~kind, msg)));
```

Comment:
This might be removed, but contains some additional information. Unfortunately I have no idea what `t` is since the name is so indescriptive.

Reply:
Ah, sorry - `t` should be `title` (some messages include a title)